plugins/modules/zos_copy.py

```diff
@@ -1404,6 +1431,9 @@ def copy_to_pdse(
         dest_members = []
 
         if src_ds_type == "USS":
+            if self.executable:
+                self.is_binary = True
```

Comment:
From what i can see it looks like you brought in the executable var into logic ```os.path.normpath("{0}/{1}".format(path, file)) if (self.is_binary or self.executable)```  so that is cleaner and  correct logic to avoid normalizing binary and exe's.